packages/peregrine/lib/talons/CartPage/PriceSummary/usePriceSummary.js

```diff
@@ -0,0 +1,32 @@
+import { useCallback, useEffect } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { useCartContext } from '@magento/peregrine/lib/context/cart';
+
+export const usePriceSummary = props => {
+    const [{ cartId }] = useCartContext();
+    const { error, loading, data } = useQuery(props.query, {
+        variables: {
+            cartId
+        },
+        fetchPolicy: 'no-cache'
```

Comment:
We should verify that actions that modify the price summary values such as shipping method or adding a shipping address elsewhere also update the price summary even if `no-cache` is used here.